toolbox.sh

```diff
@@ -0,0 +1,336 @@
+#!/usr/bin/env bash
```

Comment:
error during execution of toolbox.sh
```
   ~/git_ecocode/ecoCode-java   PR_62 ✔  ./toolbox.sh                                                                                                                            
./toolbox.sh: line 14: declare: -A: invalid option
declare: usage: declare [-afFirtx] [-p] [name[=value] ...]
Usage toolbox.sh [OPTIONS] COMMAND
Commands:
init                Initialize and creating containers
start               Starting Docker containers
stop                Stopping Docker containers
clean               Stop and remove containers, networks and volumes
build               Build the ecoCode plugin
compile             Compile the ecoCode plugin
build-docker        Build Docker services
release             Create a new release
release-push        Push the new release
Options:
-l, --logs          Display Docker container logs
-p, --push          Push the new release
-f, --force         To delete the target folder or recompile the source code
--token=<TOKEN>     Creating containers with previously created token
-h, --help          Display help
-v, --verbose       Make the command more talkative
```

Reply:
error after checking the result => no plugin deployed

Reply:
The “declare: -A: invalid option” error is linked to the use of bash on macOS, which is not recent (3.2?). The shebang line “#!/usr/bin/env bash” should have used the most recent version of bash (4.0 or higher) located in /usr/local/bin/bash.
As a precaution, a correction will be made to make the array compatible with bash v3.
The documentation for this script will be updated in the README and the ecoCode-common project today.

Reply:
ok for correction ... but the main problem is the JAR deployment to correct